include/NovelRT/Maths/GeoVector2F.h

```diff
@@ -187,7 +210,9 @@ namespace NovelRT::Maths
             *reinterpret_cast<glm::vec2*>(this) = *reinterpret_cast<const glm::vec2*>(this) / other;
             return *this;
         }
-
+        /**
+         * @brief Rotate this vector by using another point as a reference for magnitude.
+         */
```

Comment:
Sorry for all these suggestions. 😅 
```suggestion
        /**
         * @brief Rotates this vector around a given point, treating it as the origin.
         */
```

This also appears to be missing its `@param` description. Please add one.